### PR TITLE
906 dai default donation token

### PR DESCRIPTION
**Pull request by jamespfarrell**

### CODE REVIEW

**Comment by jamespfarrell** [comment]
Hey, I'm back from retreat. Yes I can look into that.

**Comment by arbreton** [comment]
Let me know if you need anything @jamespfarrell, i think is just that detail

File: src/configuration.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ const configurations = {
       useGoogleAnalytics: true,
       useHotjar: true,
     },
-    nativeTokenName: 'ETH',
+    nativeTokenName: 'DAI',
```
Comment (arbreton) [inline src/configuration.js]: Changed DAI to SAI on feathers

Comment (arbreton) [reply src/configuration.js]: Reverted change temporarly

**Comment by GriffGreen** [comment]
lets merge this!

**Comment by GriffGreen** [comment]
https://github.com/Giveth/giveth-dapp/issues/1218

**Pull request merged**